common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -18,7 +18,7 @@
     </div>
 
     <fieldset role="${input_type}group" aria-label="${label}">
```

Comment:
@ahsan-ul-haq now that we have the ```<legend>``` working so well, the role and aria-label are not needed here.  The aria-label is actually overriding the  ```<legend>``` and the role is already implied by the fieldset.